modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,205 @@
+* Existing storage mount points will remain and do not have to be touched.
+* Make sure that everything works and disable maintenance/single-user mode to put the installation back into normal operations (`occ maintance:mode --off` / `occ maintenance:singleuser --off`).
+
+=== New HTTP APIs
+
+ownCloud Server is being prepared for https://owncloud.org/news/owncloud-phoenix-rebirth-of-the-owncloud-user-interface/[Phoenix], the upcoming web frontend for ownCloud. 
+As Phoenix is separated from the backend and communicates only via HTTP APIs, it is necessary to complete the API coverage.
+
+The following new HTTP APIs have been added with Server 10.3:
+
+* WebDAV Trash bin API (https://doc.owncloud.com/server/developer_manual/webdav_api/trashbin.html[Developer documentation])
+// TODO 
+* OCS API for public link share email notifications ([Developer documentation])
+// TODO 
+* WebDAV endpoint for public links ([Developer documentation])
+https://github.com/owncloud/docs/issues/1871
+
+All new endpoints are currently in tech preview state and are mainly used for Phoenix development. 
+For this reason, they are disabled by default and have to be explicitly enabled using the new config.php option: `'dav.enable.tech_preview' => true,`.
+
+=> **Link to dev docs**
+
+=== Other notable changes
+
+* The `previews_path` config option has been added to allow customization of the thumbnail storage path (by default those reside in the user storage). https://github.com/owncloud/core/pull/35131[#35131]
+* An Activity entry is now shown when a share receiver unshares a share https://github.com/owncloud/core/pull/35193[#35193]
+* The WebUI experience on mobile devices has been improved https://github.com/owncloud/core/pull/35919[#35919] https://github.com/owncloud/core/pull/35813[#35813] https://github.com/owncloud/core/pull/35347[#35347] https://github.com/owncloud/core/pull/34803[#34803] 
+// TODO 
+* The config.php options `proxy` and `proxyuserpwd` will now be respected to enable federation when an instance needs to go through an authenticated proxy to reach a federated instance. See `config.sample.php` and the **LINK** [ownCloud Documentation] for more information. https://github.com/owncloud/core/pull/35868[#35868]
+
+https://github.com/owncloud/docs/issues/1869
+
+* The `occ files:scan` command is now case-insensitive for the userid https://github.com/owncloud/core/pull/35324[#35324]
+* A new `config.php` option (`dav.enable.tech_preview`) has been added to disable tech preview APIs by default https://github.com/owncloud/core/issues/36124[#36124]
+* [PHOENIX] Support for redirecting links to ownCloud Phoenix frontend has been added by introducing a new `config.php` option which stores the address where Phoenix is reachable (e.g., `'phoenix.baseUrl' => 'http://phoenix.example.tld:port'`) https://github.com/owncloud/core/pull/35819[#35819]
+* When loading groups, the performance of users has been improved https://github.com/owncloud/core/pull/35822[#35822]
```

Comment:
"The performance of users"? This has a different sense now, sounds wrong.

Reply:
Reverted.